charts/assets/templates/deployment.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
If we are using a persistent volume this should be a statefulset instead, see here for an example https://github.com/w3f/kusama-faucet/blob/master/charts/kusama-faucet/templates/statefulset.yaml